### PR TITLE
mail-mta/postfix: Allow writing to /etc/mail/aliases.db

### PR DESCRIPTION
**Pull request by candrews**
https://bugs.gentoo.org/show_bug.cgi?id=603458

### CODE REVIEW

**Comment by candrews** [comment]
I know revbumps are necessary, but I'm not sure which ones the maintainers want.

So please let me know which you'd like and I'll update this PR accordingly, or feel free to do so when merging on your own.

Thanks!

File: mail-mta/postfix/files/postfix.service [hunk 1]
```diff
@@ -12,8 +12,15 @@ ExecReload=/usr/sbin/postfix reload
 PrivateTmp=yes
 PrivateDevices=yes
 ProtectSystem=full
+ReadWritePaths=-/etc/mail/aliases.db
 CapabilityBoundingSet=~ CAP_NET_ADMIN CAP_SYS_ADMIN CAP_SYS_BOOT CAP_SYS_MODULE
 MemoryDenyWriteExecute=true
+ProtectKernelModules=true
```
Comment (tamiko) [inline mail-mta/postfix/files/postfix.service]: This option was introduced with systemd-232. How do older versions (especially currently stable 226-r2) behave if you use this service file?

Comment (tamiko) [inline mail-mta/postfix/files/postfix.service]: Same for the other options.

**Comment by candrews** [comment]
@tamiko systemd ignores options it doesn't know about. Ex, when systemd 226 sees `ProtectKernelModules=true` it will do nothing (no error) and continue on.

**Comment by tamiko** [comment]
@candrews Ah, thanks a lot for the clarification.

**Comment by candrews** [comment]
@tamiko can you take a look? I'm willing to make any changes that may be necessary to get this in - just let me know.

**Comment by candrews** [comment]
Separated out the additional hardening into a separate PR: https://github.com/gentoo/gentoo/pull/3629

**Pull request closed without merging**